docs/publishers/events.rst

```diff
@@ -0,0 +1,48 @@
+Custom DOM events
+=================
+
+Hypothesis client emits a few custom DOM events to which your app can react.
+
+.. option:: hypothesis:layoutchange
+
+   This event is emitted when the sidebar layout changes. Changes to the layout occur on load, when the sidebar is
+   toggled to show and hide, and when the user adjusts the sidebar manually.
+
+   The event will wrap some properties that can be used to tell the sidebar status and size.
+
+   .. option:: sidebarLayout
+
+      .. option:: expanded
+
+         ``Boolean`` If the sidebar is open, this value will be true.
+
+      .. option:: width
+
+         ``Number`` The current visible height of the sidebar.
+
+      .. option:: height
+
+         ``Number`` The current visible width of the sidebar.
+
+      .. option:: toolbarWidth
+
+         ``Number`` The current visible width of the toolbar.
+
+   .. option:: isSideBySideActive
+
+      ``Boolean`` Tells if the side-by-side mode is active.
```

Comment:
See my comments elsewhere about renaming this to `sideBySideActive`.

Reply:
I also agree with `sideBySideActive` vs. `isSideBySideActive`